Tests/test_Pathway.py

```
@@ -203,6 +204,27 @@ def test_rev(self):
+
+    def test_stochiometry(self):
+        r_1 = Reaction({"a": -1, "b": 1})
+        r_2 = Reaction({"b": -2, "c": 3})
+        system = System([r_1, r_2])
+        species, reactions, stoch = system.stochiometry()
+        self.assertEqual(species, ["a", "b", "c"])
+        self.assertCountEqual(reactions, [r_1, r_2])
+        # The order of reactions is arbitrary, so look up each row
+        # by the reaction it belongs to before checking its contents.
+        expected = {r_1: [-1, 1, 0], r_2: [0, -2, 3]}
+        for reaction, row in zip(reactions, stoch):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)